packages/promise-queue/jest.config.js

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
the babel config has a copyright and this doesn't. not sure if either of these type of files need one.